src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshService.java

```diff
@@ -67,14 +68,17 @@ public void scanMeshInboxForMessages() {
                 if (sufficientTimeRemainsInPollingCycle(pollingCycleElapsedTime)) {
                     processSingleMessage(messageId);
                 } else {
-                    LOGGER.warn("Insufficient time remains to complete the polling cycle. Processed {} of {} messages from inbox.", i + 1, inboxMessageIds.size());
+                    LOGGER.warn("Insufficient time remains to complete the polling cycle. Processed {} of {} messages from inbox.",
+                        i + 1, inboxMessageIds.size());
                     return;
                 }
             }
             LOGGER.info("Completed MESH mailbox polling cycle. Processed all messages from inbox.");
         } else {
-            LOGGER.info("Could not obtain database lock to run MESH mailbox polling cycle: insufficient time has elapsed " +
-                    "since the previous polling cycle or another adaptor instance has already started the polling cycle. " +
+            LOGGER.info("Could not obtain database lock to run MESH mailbox polling cycle: insufficient time has elapsed "
+                +
+                    "since the previous polling cycle or another adaptor instance has already started the polling cycle. "
+                +
                     "Next scan in {} seconds", TimeUnit.SECONDS.convert(wakeupIntervalInMilliseconds, TimeUnit.MILLISECONDS));
```

Comment:
```suggestion
            LOGGER.info("Could not obtain database lock to run MESH mailbox polling cycle: insufficient time has elapsed "
                + "since the previous polling cycle or another adaptor instance has already started the polling cycle. "
                + "Next scan in {} seconds", TimeUnit.SECONDS.convert(wakeupIntervalInMilliseconds, TimeUnit.MILLISECONDS));
```

Reply:
Done in 0d2ce4bcf1